tuned/utils/profile_recommender.py

```diff
@@ -32,7 +32,13 @@ def recommend(self, hardcoded = False):
 		profile = consts.DEFAULT_PROFILE
 		if hardcoded:
 			return profile
-		matching = self.process_config(consts.RECOMMEND_CONF_FILE)
+		has_root = os.getuid() == 0
```

Comment:
This should use `os.geteuid()`. The effective UID matters, not the real one.

Reply:
@yarda Can you please fix that up once you merge this? I fixed one more new occurence of the problem here https://github.com/redhat-performance/tuned/pull/172

Reply:
Fixed.